Toluene/Burcat 1986 yaml files/Burcat_mixA.yaml

```diff
@@ -0,0 +1,149 @@
+---
+file-author:
+    name: Morgan Mayer
+    ORCID: 0000-0001-7137-5721
+file-version: 0
+chemked-version: 0.3.0
+reference:
+    doi:

```

Comment:
I think we should remove fields that are blank from the files. IIRC, this should actually raise a validation error...